src/include/miopen/tensor_layout.hpp

```diff
@@ -35,6 +35,26 @@
 
 namespace miopen {
 
+enum tensor_layout_t
```

Comment:
This enum is a duplicate of `miopenTensorLayout_t` in `miopen.h`, so let's remove it and use `miopenTensorLayout_t`. You can split that as a separate PR that merges before this PR since the scopes are different.

Reply:
Sure, it makes perfect sense to do that. However, these tests do not use `tensor_layout_t` directly, so there's no need to connect that work with this PR. Just let me know when to pull it into a sprint; it would also be a good task for @BrianHarrisonAMD.

Reply:
@randyspauldingamd : I am fine with creating a follow up PR and task for it but please assign it to yourself. @iq136boy will work with @BrianHarrisonAMD to figure out what he'll work on next. CC: @JehandadKhan